LayoutTests/svg/clip-path/clip-path-use-referencing-clipped-text-expected.html

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
+<div style="width: 100px; height: 100px; background-color: green"></div>
```

Comment:
I think we may have an existing html file featuring a green 100x100 rect.

Reply:
Last time I checked there was no mechanism to re-use files named differently, other than symlinks.

Reply:
SVG WPT tests use "<link rel="match" ...", is it possible to use that?